File: app/src/main/java/qing/com/kotlin3/Main2Activity.java
package qing.com.kotlin3;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Toast;

import com.posun.lightui.recyclerview.LightFormAdapterManager;

import qing.com.kotlin3.test.TestGradAdapter;
import qing.com.kotlin3.test.TestGroupAdapter;
import qing.com.kotlin3.test.TestListAdapter;

public class Main2Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.recyclerview);
        final LightFormAdapterManager adapterManager = new LightFormAdapterManager(this);
        View titleView = getLayoutInflater().inflate(R.layout.fix_uilayout, null);
        LightFormAdapterManager.LightRecyclerFixedAdapter fixedAdapter = new LightFormAdapterManager.LightRecyclerFixedAdapter(titleView);
        final TestGradAdapter testGradAdapter = new TestGradAdapter(3);
        /***********************************************/
        adapterManager.addAdapter(fixedAdapter);
        adapterManager.addAdapter(testGradAdapter);
        adapterManager.addAdapter(new TestGradAdapter(4));
        adapterManager.addAdapter(new TestListAdapter());
        adapterManager.addAdapter(new TestGroupAdapter());

        adapterManager.setRecyclerView(recyclerView);
        /**********************************************************/


        testGradAdapter.setOnItemCilckListener(new LightFormAdapterManager.OnItemCilckListener() {
            @Override
            public void onItemClick(int position) {
                Toast.makeText(Main2Activity.this, "OnItemCilckListener ="+position, Toast.LENGTH_SHORT).show();
                int cont = testGradAdapter.getCont();
                testGradAdapter.setCont(cont + 1);
                adapterManager.notifyChildItemRangeInserted(cont,1,testGradAdapter);
            }

            @Override
            public void onLongItemClick(int position) {

            }
        });
    }
}

File: lbsx64/src/main/java/lightplugin/posun/com/lbsx64/Main2Activity.kt
package lightplugin.posun.com.lbsx64

import android.content.Intent
import android.os.Build
import android.os.Bundle
import android.support.annotation.RequiresApi
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.widget.Toast
import com.qing.lighthttp.LightHandlerThread
import kotlinx.android.synthetic.main.activity_main2.*

class Main2Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)
        hello.setOnClickListener {
            //            Toast.makeText(this, "111111", Toast.LENGTH_SHORT).show()
            LightHandlerThread.postMsgToMainThread("123456", this::test)
        }
    }

    fun test(string: String) {
        Toast.makeText(this, string, Toast.LENGTH_SHORT).show()
    }
    @RequiresApi(Build.VERSION_CODES.M)
    fun test(string: String, item:String) {
//        Toast.makeText(this, string, Toast.LENGTH_SHORT).show()

        val intent = Intent(this, MainActivity::class.java)
//        FMTest().with(this).startforResult(intent) {
//        string-> Log.i("tag", string as String)}
    }
}
